### ISSUE TITLE
Adding a few more character expressions to the current EmpathicLC code

### ISSUE THREAD

**Issue by IvonArroyo** [center-for-knowledge-communication/mathspring#432]
Add the following character behaviors to EmotionalLC.java  or FullEmpathicLC.java (check where it would be) to do the following:

- Video.html
- Example.html

When the student makes an incorrect attempt, the first time, the hint flashes. The second time, there is a message from the character. That second time, it sometimes offers a hint or alike (talking).

I want to add to the list of messages in that situation  (2nd incorrect) that video.html is shown IF there is a video available for that problem, and only if there is one.

I also want to add to the list of possible messages in that situation  (2nd incorrect) that example.html is shown IF there is an example available for that problem, and only if there is one.

Note there is some logic in there so that the same messages don't repeat too often.

SEcond, it seems that  the following messages are not being shown (I believe EmotionalLC.java)? Please double check, they should be in some list of messages and eligible in some situations. If they are missing, please add them.

effortAttribution1.html
effortAttribution2.html

Thanks!

**Comment by FrankSylvia** [comment]
I have implemented this using interventions.  The sequence now is:
    On the first wrong answer, highlight the hint button, if student hasn't already seen a hint.
    On the second wrong answer, the showVideo function is run, if the is a video for this problem.
    On the third wrong answer, show an example, if there is an example for this topic.
    
    I saw that EmpathicLC.java is executing the HighlightHint intervention.  I could have put the logic for doing all three interventions in there, but I thought the intervention approach was more in keeping with the MS design approach.  I caouldchange it if you want me to.
    
I saw no references to these html files anywhere in Mathspring.  Do these files exist somewhere?  
effortAttribution1.html
effortAttribution2.html

Please advise on there questions.

**Comment by IvonArroyo** [comment]
Just add this, and see if the example or video (messages of the character) show up after the second incorrect response.

  public static final String[] incorrectStrategic = {
        "incorrect1",
        "incorrect2",
        "incorrect3",
        "incorrect4",
        "example",
        "video"
    } ;

**Comment by IvonArroyo** [comment]
I tested this today on the dev server, and what happens is that they do show up eventually, but clicking on the example and video buttons tells me there are NO video and NO example, at the moment when the example.html or video.html showed up.

See attached screenshots, after I clicked the example/video buttons, after the character shows example/video messages.

TIP:  could we move "example" and "video" to the top of the array (positions 0 and 1)? That would make it easier for them to show up. Apparently it starts from the top of the list.
<img width="1592" alt="Screen Shot 2021-05-21 at 4 06 33 PM" src="https://user-images.githubusercontent.com/5631787/119192960-eceb6f80-ba4e-11eb-83c1-0772270c669d.png">
<img width="1393" alt="Screen Shot 2021-05-21 at 4 03 10 PM" src="https://user-images.githubusercontent.com/5631787/119193001-f8d73180-ba4e-11eb-87ed-4ef40456f028.png">
<img width="1306" alt="Screen Shot 2021-05-21 at 4 03 27 PM" src="https://user-images.githubusercontent.com/5631787/119193006-fbd22200-ba4e-11eb-9a59-359a7c2451f7.png">
